backend/blender_script/main.py

```diff
@@ -2,12 +2,61 @@
 import random
 import sys
 import os
+from math import radians
 # Add the directory containing the 'lens' module to the sys.path
-bpy.ops.wm.read_factory_settings(use_empty=True)
+#bpy.ops.wm.read_factory_settings(use_empty=True)
 module_dir = os.path.dirname(__file__)
 sys.path.append(module_dir)
 from lens import Lens, LensPrescription, Prescription
 
+def insetLens():
+
+    rightLens = bpy.data.objects['Lens_1']
+    leftLens = bpy.data.objects['Lens_2']
+
+    rightLens.location = bpy.data.objects['Right_Eye_Location'].location
+    leftLens.location = bpy.data.objects['Left_Eye_Location'].location
+
+    #rightLens.rotation_euler = bpy.data.objects['Right_Eye_Location'].rotation_euler.copy()
+    rightLens.rotation_euler.x = bpy.data.objects['Right_Eye_Location'].rotation_euler.x 
+    rightLens.rotation_euler.y = bpy.data.objects['Right_Eye_Location'].rotation_euler.y 
+    rightLens.rotation_euler.z = bpy.data.objects['Right_Eye_Location'].rotation_euler.z - radians(90)
+    #leftLens.rotation_euler = bpy.data.objects['Left_Eye_Location'].rotation_euler.copy()
+    leftLens.rotation_euler.x = bpy.data.objects['Left_Eye_Location'].rotation_euler.x 
+    leftLens.rotation_euler.y = bpy.data.objects['Left_Eye_Location'].rotation_euler.y 
+    leftLens.rotation_euler.z = bpy.data.objects['Left_Eye_Location'].rotation_euler.z - radians(90)
```

Comment:
@EsteBran these are the lines I'm talking about, for some reason they don't get copied. I've tried a few different methods, this was the last

Reply:
If you want something less wordy then you could try something like this

```
rotation = bpy.data.objects['Right_Eye_Location'].rotation_euler
rightLens.rotation_euler = (rotation.x, rotation.y, rotation.z - radians(90))  
```
Either tuple or array notation works.
It should work, since rotation.x is supported in the blender python console

Reply:
Yea i tried something similar and it still didn't work. trying this also didn't fix it
This is the empties the lenses should follow (ive exaggerated the rotations here to test)


Lenses should be facing downward
![image](https://github.com/Lenscrafters-Capstone/LensCrafters-free/assets/15898988/505f9405-bc4b-4f11-aea3-d23271f3061d)

But this is the result
![image](https://github.com/Lenscrafters-Capstone/LensCrafters-free/assets/15898988/1ad52dca-be32-4432-964f-8ef46c086f42)

Reply:
Hmm. What happens if you print the value of rotation_euler?

As a last resort you could hardcode the rotation. It shouldn't be too bad since we generally generate and import on pretty well defined axes. For example lenses are always generated at (0,0,0) facing +X.  I think frames are imported such that they face +Y or -Y, not sure.

Reply:
Oh one more thing to keep in mind, I'm pretty sure every time you rotate something (or modify something), that object has to be the active selected object. One way I did that in my cylinder generation was by this:

```
bpy.context.view_layer.objects.active = obj
```

Maybe try that?